src/NuGet.LibraryModel/project.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+    "version": "1.0.0-*",
+    "dependencies": {
+        "Newtonsoft.Json": "6.0.6",
+        "NuGet.Versioning": "1.0.0-*"
```

Comment:
Note: When the runtime uses this, it will be pulling it and NuGet.Versioning in as source, so this project.json will be irrelevant.